src/components/EventToast/useEventToast.js

```diff
@@ -8,39 +7,25 @@ import { isFuture, parse } from "date-fns";
 import { isHomeSite } from "lib/legacy";
 import { useChainId } from "lib/chains";
 import { useMarketsInfo } from "domain/synthetics/markets";
-import { useOracleKeeperFetcher } from "domain/synthetics/tokens";
-import { ARBITRUM } from "config/chains";
+import useIncentiveStats from "domain/synthetics/common/useIncentiveStats";
 
 function useEventToast() {
   const isHome = isHomeSite();
   const [visited, setVisited] = useLocalStorage("visited-announcements", []);
   const { chainId } = useChainId();
   const { marketsInfoData } = useMarketsInfo(chainId);
-  const [isIncentivesActive, setIsIncentivesActive] = useState(false);
+  const incentiveStats = useIncentiveStats();
```

Comment:
before the change it always requested data for ARBITRUM. the notification should be displayed for all networks

Reply:
i think useIncentiveStats can accept `chainId`, and inside useEventToast it can be called as `useIncentiveStats(ARBITRUM)`

Reply:
Updated!